docs/manual/determinism-checker.tex

```diff
@@ -448,9 +448,9 @@
 myMethod(detList , index);
 \end{Verbatim}
 
-\subsectionAndLabel{Disallowing \<OrderNonDet> in polymorphic instantiations}{determinism-no-ordernondet}
+\subsectionAndLabel{Disallowing \<@OrderNonDet> in polymorphic instantiations}{determinism-no-ordernondet}
```

Comment:
Could you please push the unrelated manual changes to the main branch so that they don't clutter the diffs here?